README.md

```diff
@@ -1,59 +1,39 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+[](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
 
-> One paragraph statement about the project.
+> Mobile portfolio page
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+This is my Portfolio
```

Comment:
- [ ] For this and the next projects you should add a better description, just try to explain what you did, 
what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description 😉).